src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/TimeSpanConverter.cs

```diff
@@ -91,5 +93,12 @@ internal override void WriteAsPropertyNameCore(Utf8JsonWriter writer, TimeSpan v
 
             writer.WritePropertyName(output.Slice(0, bytesWritten));
         }
+
+        internal override JsonSchema? GetSchema(JsonNumberHandling _) => new()
+            {
+                Type = JsonSchemaType.String,
+                // TimeSpan is represented as a string in the format "[-][d.]hh:mm:ss[.fffffff]"
+                Pattern = @"^-?(\d+\.)?\d{2}:\d{2}:\d{2}(\.\d{1,7})?$"
```

Comment:
These regexes end up in the generated schema but we're not actually processing them anywhere, right?

Reply:
They're being evaluated by JsonSchema.NET in the tests.

Reply:
Right, I meant we're not processing them as part of the src.

Reply:
Correct.